charts/backingservices/charts/constellation/README.md

```diff
@@ -1,43 +1,49 @@
-
-This minimal setup has the restriction that custom images, custom components and localisations cannot be added to the application. The next step is the app-static service that handles the customer application specific content. The app-static service can be setup using the ConstellationUI helm chart described below.
+for instructions related to the pegastatic content delivery setup. Once that is complete please proceed with the instructions below for the constellation appstatic service setup.
 
 ## ConstellationUI helm chart
 
-The ConstellationUI helm chart is used to deploy an instance of app-static service into a Kubernetes environment. The service is capable of serving multiple client environments. We encourage having a single deployment for all your systems. The following readme provides a detailed description of the parameter configurations and their default values as applicable. 
+The ConstellationUI helm chart is used to deploy an instance of the constellation appstatic service into a Kubernetes environment. The service is capable of serving multiple client environments. We encourage having a single deployment for all your systems. The following readme provides a detailed description of the parameter configurations and their default values as applicable. 
 
 #### Prerequisites
 #### AWS Cloud 
-1. The ConstellationUI helm chart for AWS provider uses an application load balancer to expose the service. Before installing the constellationui helm chart please follow the prerequisites section of the following AWS documentation and make sure your cluster is configured accordingly. 
+1. The ConstellationUI helm charts for AWS provider use an application load balancer to expose the service. Before installing the constellationui helm chart please follow the prerequisites section of the following AWS documentation and make sure your cluster is configured accordingly. 
 
 2. If you are using a custom domain please make sure you have a valid certificate imported or created in ACM.
 
 > https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html
 
+#### Downloading Docker images for your deployment
+
+Clients with appropriate licenses can request access to several required images from the Pega-managed Docker image repository. With your access key, you can log in to the image repository and download these Docker images to install the constellation appstatic service.
+
+To download the constellation appstatic service image, specify the version and use the following command
+
+```bash
+$ sudo docker pull pega-docker.downloads.pega.com/constellation-appstatic-service/docker-image:xxxxxxx
+
+Digest: <encryption verification>
+Status: Downloaded pega-docker.downloads.pega.com/constellation-appstatic-service/docker-image:xxxxxxx
+```
+
+For details about downloading and then pushing Docker images to your repository for your deployment, see [Using Pega-provided Docker images](https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images).
```

Comment:
this is the correct link to the doc now>>
[Pega-provided Docker images](https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html)

Reply:
@taz-pega-work Can you please review and approve?